src/bug-filing/azure-devops/azure-devops-bug-filing-service.tsx

```diff
@@ -29,21 +33,17 @@ function getSettingsFromStoreData(bugServicePropertiesMap: BugServicePropertiesM
 }
 
 function isSettingsValid(data: AzureDevOpsBugFilingSettings): boolean {
-    return !isEmpty(data) && isStringValid(data.projectURL) && isStringValid(data.issueDetailsLocationField);
+    return !isEmpty(data) && isStringValid(data.projectURL) && isStringValid(data.issueDetailsField);
 }
 
 function isStringValid(stringToCheck: string): boolean {
     return !isEmpty(stringToCheck) && !isEmpty(stringToCheck.trim());
 }
 
-const settingsForm = NamedSFC<SettingsFormProps<AzureDevOpsBugFilingSettings>>('BugFilingSettings', props => {
-    return <div />;
-});
-
 export const AzureDevOpsBugFilingService: BugFilingService<AzureDevOpsBugFilingSettings> = {
     key: AzureDevOpsBugFilingServiceKey,
-    displayName: 'AzureDevOps',
-    settingsForm,
+    displayName: 'Azure DevOps',
```

Comment:
I believe we should be using Azure Boards as the brand instead of Azure DevOps (ideally both in UI and code)
```suggestion
    displayName: 'Azure Boards',
```